src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -906,6 +907,10 @@ namespace ReportSizingManager {
 									TotCapTempModFac = CurveValue ( DataTotCapCurveIndex, CoilInWetBulb, OutTemp );
 								} else {
 									TotCapTempModFac = 1.0;
+									
+									if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
+										EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, TotCapTempModFac);
```

Comment:
I'll pull this branch and fix this in a few hours.

On 9/23/2015 1:41 AM, Rongpeng Zhang wrote:
>
> In src/EnergyPlus/ReportSizingManager.cc 
> <https://github.com/NREL/EnergyPlus/pull/5210#discussion_r40172345>:
>
> > @@ -906,6 +907,10 @@ namespace ReportSizingManager {
> >  									TotCapTempModFac = CurveValue ( DataTotCapCurveIndex, CoilInWetBulb, OutTemp );
> >  								} else {
> >  									TotCapTempModFac = 1.0;
> > +									
> > +									if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
> > +										EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, TotCapTempModFac);
>
> @rraustad <https://github.com/rraustad> @mjwitte 
> <https://github.com/mjwitte>
>
> It seems we cannot use the method that Rich suggested, i.e., to 
> calculate *DataTotCapCurveValue* with the following codes in 
> /DXCoils.cc/, and then directly use it to assign *TotCapTempModFac* in 
> /ReportSizingManager.cc/.
>
> EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, DataTotCapCurveValue);
>
> The problem is that we need air condition at sizing (e.g., 
> CoilInWetBulb and CoilInTemp), which can only be obtained within 
> /ReportSizingManager.cc/
>
> ==========This is what I coded in*ReportSizingManager.cc*==========
>
>      if ( DataTotCapCurveIndex > 0 ) {
>          TotCapTempModFac =*CurveValue*  ( DataTotCapCurveIndex,*CoilInWetBulb*  , OutTemp );
>      } else {
>          TotCapTempModFac = 1.0;
>          
>          if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
>              *CalcVRFCoilCapModFac*( 0, _, CompName,*CoilInTemp*, _, _, _, TotCapTempModFac);
>          }
>      }
>
> If there is no better solution, I think it is fine to use the current 
> design, right? We have to call a method to calculate 
> */TotCapTempModFac */ anyway: current VRF coil uses /CurveValue/ 
> method , while the new VRF coil uses a specifically designed method 
> /CalcVRFCoilCapModFac/.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NREL/EnergyPlus/pull/5210/files#r40172345>.
>

-- 
Richard Raustad
Florida Solar Energy Center
1679 Clearlake Road
Cocoa, FL 32922
Ph: (321)638-1454
http://www.fsec.ucf.edu/en/

Program Director
Electric Vehicle Transportation Center
http://evtc.fsec.ucf.edu/